content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -158,7 +158,7 @@ def requested_storage(args_hash)
     end
     if args_hash[:resource].options[:disk_remove]
       args_hash[:resource].options[:disk_remove].each do |disk|
-        disk_num = disk[:disk_name].match(/_(\d).vmdk/)
+        disk_num = disk['disk_name'].match(/_(\d).vmdk/)
```

Comment:
@billfitzgerald0120 Is this a symbol versus string issue, has it worked before with a symbol?
Do we have spec for this.

Reply:
@mkanoor @tinaafitz  I re-created on 5.8.4.5 using the rest-api.  The UI worked with the original code but not when using the rest-api.   I tested this on master with the same results.  There is a requested_spec but it doesn't test the disk remove.

Reply:
@billfitzgerald0120 Is the spec file missing from this PR?

Reply:
@mkanoor I just added the spec file.